mrtt-ui/src/views/SiteQuestionsOverview/AddMonitoringSectionMenu.js

```diff
@@ -25,10 +25,9 @@ const AddMonitoringSectionMenu = () => {
         {language.pages.siteQuestionsOverview.addMonitoringSectionButton}
         <ArrowDropDown />
       </ButtonPrimary>
-
       <Menu
         id='add-monitoring-section-menu'
-        anchorElement={anchorElement}
+        anchorEl={menuAnchorElement}
```

Comment:
the Menu prop was supposed to be `anchorEl` and somewhere along the way I probably accidentally edited it with a multi cursor in vscode (crtl + shift + L )